main.py

```diff
@@ -246,47 +245,87 @@ def game_loop():
     mixer.music.play(-1)
     #mixer.music.set_volume(0)
 
-async def update_enemy(game_client,local_player,enemy_character):
+
+async def update_enemy(game_client, local_player, enemy_character):
     for message in game_client.get_updates():
         local_player.health = message.enemyHealth
         enemy_character.move_enemy(SCREEN_WIDTH, SCREEN_HEIGHT, screen, local_player, obstacles,
 
                                    message.keys, message.x, message.y)
+
+
 def run_once(loop):
     loop.call_soon(loop.stop)
     loop.run_forever()
 
+
 def multi_player_game_loop(game_client):
-    #gameKeys = [pygame.K_a, pygame.K_d, pygame.K_w, pygame.K_r, pygame.K_t]
+    # gameKeys = [pygame.K_a, pygame.K_d, pygame.K_w, pygame.K_r, pygame.K_t]
     pygame.init()
+    if game_client.map_choice == 1:
```

Comment:
would it save you doing 3 if statements if you set game_client.map_choice to the path of the map instead of a number and just said map_chosen = game_client.map_choice

Reply:
we would still need them for loading in the obstacles I think.